### PR TITLE
Ref [FEATURE][51921] Update feature creator screen, create basic detail creator screen

### PR DESCRIPTION
**Pull request by minhnn-1279**
[Feature #51921: Creator Screen](https://edu-redmine.sun-asterisk.vn/issues/51921)

What:
- Creator Screen
- Fetch data from API, Create Adapter
- Handle Event

image
image

### CODE REVIEW

File: app/src/main/java/com/nguyennhatminh614/marvelapp/util/DTOItemAdapter.kt [hunk 1]
```diff
@@ -9,7 +9,7 @@ import com.nguyennhatminh614.marvelapp.databinding.LayoutDtoItemBinding
 class DTOItemAdapter<T : DtoItem> :
     RecyclerView.Adapter<DTOItemAdapter<T>.ViewHolder>() {
 
-    private val listDTOItem = mutableListOf<T>()
+    private var listDTOItem = mutableListOf<T>()
     private var clickItemInterface: OnClickItemInterface<T>? = null
```
Comment (cuong117) [inline app/src/main/java/com/nguyennhatminh614/marvelapp/util/DTOItemAdapter.kt:13]: nên để val

File: app/src/main/res/layout/fragment_drawer_creator.xml [hunk 2]
```diff
@@ -1,6 +1,34 @@
...
 
+    <TextView
+        android:id="@+id/bigTitle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:fontFamily="@font/comfortaa_bold"
+        android:text="@string/creator"
+        android:textSize="@dimen/title_text_size"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
```
Comment (cuongvd-1287) [inline app/src/main/res/layout/fragment_drawer_creator.xml:15]: sắp xếp lại nhá: [https://github.com/framgia/coding-standards/blob/master/vn/android/coding_convention_android_kotlin.md#33-attributes-ordering](url)

**Review by ghost — APPROVED** [review]

File: app/src/main/java/com/nguyennhatminh614/marvelapp/screen/creator/CreatorAdapter.kt [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+import com.nguyennhatminh614.marvelapp.util.extensions.loadGlideImageFromUrl
+
+class CreatorAdapter : RecyclerView.Adapter<CreatorAdapter.ViewHolder>() {
+    private val listCreator = mutableListOf<Creator>()
+    private var binding: CreatorItemLayoutBinding? = null
+    private var clickItemInterface: OnClickItemInterface<Creator>? = null
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
+        binding = CreatorItemLayoutBinding.inflate(LayoutInflater.from(parent.context), parent, false)
+        return ViewHolder(binding!!)
```
Comment (daolq-2712) [inline app/src/main/java/com/nguyennhatminh614/marvelapp/screen/creator/CreatorAdapter.kt]: không force unwrap @minhnn-1279

File: app/src/main/java/com/nguyennhatminh614/marvelapp/screen/creator/CreatorContract.kt [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+class CreatorContract {
+    interface View {
+        fun onSuccess(listCreator: MutableList<Creator>?)
+        fun onError(exception: Exception?)
+    }
+
+    interface Presenter {
+        fun getCreatorListRemote()
+    }
+
```
Comment (daolq-2712) [inline app/src/main/java/com/nguyennhatminh614/marvelapp/screen/creator/CreatorContract.kt]:
```suggestion
```

File: app/src/main/java/com/nguyennhatminh614/marvelapp/screen/creator/DetailCreatorFragment.kt [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+            this.creator = creator
+        }
+    }
+
+    private fun <T> checkList(
+        list: MutableList<T>,
+        notEmptyListEvent: (MutableList<T>) -> Unit,
+        emptyListEvent: () -> Unit
+    ) {
+
```
Comment (daolq-2712) [inline app/src/main/java/com/nguyennhatminh614/marvelapp/screen/creator/DetailCreatorFragment.kt]:
```suggestion
```

**Review by daolq-2712 — COMMENTED** [review]
Pls check my comments!

**Pull request merged**